Bibliotheque nationale de France.js

```diff
@@ -565,7 +596,10 @@ var BnfClass = function() {
 					}
 				}
 			}
-				
+			// case last zone
+			if(tag) {
+						record.addField(tag, ind, content);
+					}	
```

Comment:
Huch, okay, that might be a really strange logic in this loop. Because in each step there will be tried to output the previous data with `record.addField(tag, ind, content);` and then fill these values with the current data... I understand now, but I will forget about this in some weeks...